cmd/podman/root.go

```diff
@@ -540,7 +540,7 @@ func rootFlags(cmd *cobra.Command, podmanConfig *entities.PodmanConfig) {
 		_ = pFlags.MarkHidden(networkBackendFlagName)
 
 		rootFlagName := "root"
-		pFlags.StringVar(&podmanConfig.ContainersConf.Engine.StaticDir, rootFlagName, podmanConfig.ContainersConfDefaultsRO.Engine.StaticDir, "Path to the root directory in which data, including images, is stored")
+		pFlags.StringVar(&podmanConfig.GraphRoot, rootFlagName, "", "Path to the graph root directory where images, containers, etc. are stored")
```

Comment:
Are we concerned about this breaking folks? Setting `--root` is now doing something very different than it did before.

Reply:
I honestly don't know.  This stuff is really broken :(

Reply:
I think we're good to go.  It essentially overrode the graph root before as well (that's how it was being used).